### PR TITLE
Yf get reference properly threaded through

### PR DESCRIPTION
**Pull request by yfarjoun**
not sure how this happened...but it seems that the code was looking at the wrong version of the reference...

### CODE REVIEW

File: src/main/java/picard/fingerprint/ExtractFingerprint.java [hunk 1]
```diff
@@ -97,7 +97,7 @@ protected int doWork() {
         }
 
         checker.setLocusMaxReads(LOCUS_MAX_READS);
-        checker.setReferenceFasta(REFERENCE_SEQUENCE);
+        checker.setReferenceFasta(referenceSequence.getReferenceFile());
```
Comment (gbggrant) [inline src/main/java/picard/fingerprint/ExtractFingerprint.java:100]: Isn't parseArgs doing this?https://github.com/broadinstitute/picard/blob/4c937e5bb8b790a5b7096c1e61564633e83298f0/src/main/java/picard/cmdline/CommandLineProgram.java#L359
(or are you removing deprecated usage?)

**Review by gbggrant — APPROVED** [review]
LGTM.  Not clear why this change is needed.

File: src/main/java/picard/fingerprint/ExtractFingerprint.java (hunk 1 above)
Comment (yfarjoun) [reply src/main/java/picard/fingerprint/ExtractFingerprint.java:100]: I got a report that without this fix the tool was unable to open cram files. I didn't verify this claim though...so I should probably look at the tests....

**Comment by yfarjoun** [comment]
yeah....this works in master...sorry for the trouble.

**Pull request closed without merging**